app/screens/home/search/search.tsx

```diff
@@ -68,6 +68,9 @@ const styles = StyleSheet.create({
 const getSearchParams = (terms: string, filterValue?: FileFilter) => {
     const fileExtensions = filterFileExtensions(filterValue);
     const extensionTerms = fileExtensions ? ' ' + fileExtensions : '';
+    
+    terms = terms.replace("@","from:")
```

Comment:
This will search all the messages from `from:<username>` if someone types `@<username>`, please let me know if it is the expected behavior, what if someone wants to search occurrences of `@<username>`?

Reply:
This does not seem to match the work being done in https://github.com/mattermost/mattermost/pull/24168

I think is best if you guys coordinate along side with @esethna as acting PM so that the behavior is exactly the same in mobile and in the webapp

Reply:
Understood what needs to be done. I'll update it by today

Reply:
@johndavidlugtu @enahum  I have made the changes